pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationProviderToken.java

```diff
@@ -126,9 +137,40 @@ private static String validateToken(final String token) throws AuthenticationExc
     @SuppressWarnings("unchecked")
     private Jwt<?, Claims> authenticateToken(final String token) throws AuthenticationException {
         try {
-            return Jwts.parser()
+            Jwt<?, Claims> jwt = Jwts.parser()
                     .setSigningKey(validationKey)
                     .parse(token);
+
+            if (audienceClaim != null) {
+                if (audience == null){
```

Comment:
I think we need to move the configuration validation logic to right after line 86. Since it indicates iellgall configuration settings.

Reply:
I think the check here should be moved to `initialize` method.